app/api/helpers/scheduled_jobs.py

```diff
@@ -142,10 +142,10 @@ def send_event_fee_notification_followup():
                                                         incomplete_invoice.event.id)
 
 
-def expire_pending_tickets_after_three_days():
+def expire_pending_tickets():
     from app import current_app as app
     with app.app_context():
         db.session.query(Order).filter(Order.status == 'pending',
-                                       (Order.created_at + datetime.timedelta(days=3)) <= datetime.datetime.now()).\
+                                       (Order.created_at + datetime.timedelta(minutes=30)) <= datetime.datetime.now()).\
```

Comment:
Still 30

Reply:
@iamareebjamal Actually, we are checking for the tickets which are expiring after 30mins using a cron which is scheduled for every 45 minutes

Reply:
@mrsaicharan1 I didn't follow your comment here. I suggested that `order.created_at + 45minutes` should be used here. 
Cron job determines frequency of the check only. It should be 15 minutes IMO

Reply:
> @mrsaicharan1 I didn't follow your comment here. I suggested that `order.created_at + 45minutes` should be used here.
> Cron job determines frequency of the check only. It should be 15 minutes IMO

@shreyanshdwivedi I guess the cron frequency is independent. So basically, to get it straight, the form filling is separate from the payment completion right? I took it as a whole. Will update now.